server/websocket_manager.py

```
@@ -3,32 +3,94 @@
 import redis.asyncio as redis
 import asyncio
 import json
+import uuid
```

Ruff (0.12.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)